app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -544,6 +364,48 @@ class TopicFragmentTest {
     return launch(createTopicActivityIntent(internalProfileId, topicId))
   }
 
+  private fun clickTabAtPosition(position: Int) {
+    onView(
+      Matchers.allOf(
+        withText(TopicTab.getTabForPosition(position).name),
+        isDescendantOfA(withId(R.id.topic_tabs_container))
+      )
+    ).perform(click())
+  }
+
+  private fun matchTabTitleAtPosition(position: Int) {
```

Comment:
Here & below: suggest phrasing this in terms of making assertions. For example: ``verifyTabTitleAtPosition`` ('verify' is stronger language than just 'match' since it implies we need it to be a certain state).

Reply:
Sounds good, Done